pyclient/deephaven/qst.py

```diff
@@ -0,0 +1,91 @@
+from abc import ABC, abstractmethod
+from enum import Enum
+
+
+class TableOp(ABC):
+    def __init__(self, parent=None):
+        self.parent = parent
+        if parent:
+            self.parent.child = self
+        self.child = None
+
+    def accept(self, visitor):
```

Comment:
In java, this is an abstract method - each concrete impl is response for implementing. Given that overloading is a bit different though, I might except each concrete impl calls a method based on its own type:

```python
class UpdateOp(TableOp):
    def accept(self, visitor):
      visitor.visitUpdate(self)
      
class ViewOp(TableOp):
    def accept(self, visitor):
      visitor.visitView(self)
...
```